docs/dev/testing.md

```diff
@@ -0,0 +1,34 @@
+# Running the tests
```

Comment:
Can we add a note for Windows users (see [my comment](https://github.com/mozilla/glean/pull/117#discussion_r290659092)) or file a follow-up for adding it?